app/templates/macros/page_macros.html

```diff
@@ -23,7 +23,7 @@
         {% if current_user.is_admin() %}
```

Comment:
if all of the buttons and update alert has `style="display:none"` we don't need this since they are initially all hidden by that styling so you can remove this entire `{% if %}` block